invenio_github/api.py

```diff
@@ -243,13 +259,14 @@ def sync_repo_hook(self, repo_id):
         # If hook on GitHub exists, get or create corresponding db object and
         # enable the hook. Otherwise remove the old hook information.
         repo = Repository.get(repo_id, gh_repo.full_name)
-        if not repo:
-            repo = Repository.create(self.user_id, repo_id, gh_repo.full_name)
 
         if hook:
+            if not repo:
```

Comment:
This means that we were creating one local repo per repository that users had in `remote_account.extra_data`. Therefore, we have too many repos now created locally.

There is no big impact but we can clean this up. I created an [issue](https://github.com/zenodo/rdm-project/issues/454) for that

Reply:
could you add a comment in the code?